config/routes.rb

```diff
@@ -5,7 +5,9 @@
     mount LetterOpenerWeb::Engine, at: "/letter_opener"
   end
 
-  devise_for :personalities
+  devise_for :personalities, controllers: {
+      sessions: "admin/devise/sessions"
```

Comment:
__[RuboCop]__

Style/TrailingCommaInLiteral: Put a comma after the last item of a multiline hash.


<!-- Comment by [SideCI](https://sideci.com) -->